pkg/kubelet/cm/container_manager_stub.go

```diff
@@ -125,7 +125,7 @@ func (cm *containerManagerStub) UpdatePluginResources(*schedulerframework.NodeIn
 }
 
 func (cm *containerManagerStub) InternalContainerLifecycle() InternalContainerLifecycle {
-	return &internalContainerLifecycleImpl{cpumanager.NewFakeManager(), memorymanager.NewFakeManager(), topologymanager.NewFakeManager()}
+	return &internalContainerLifecycleImpl{cpumanager.NewFakeManager(klog.Background()), memorymanager.NewFakeManager(), topologymanager.NewFakeManager()}
```

Comment:
The same here.

Reply:
Here I think we can just save the context being passed and reuse here. It's functionally equivalent.